File: customer/src/main/java/com/lexuancong/customer/viewmodel/address/AddressDetailVm.java
package com.lexuancong.customer.viewmodel.address;

public record AddressDetailVm(
        Long id,
        String contactName,
        String phoneNumber,
        String specificAddress,
        Long districtId,
        String districtName,
        Long provinceId,
        String provinceName,
        Long countryId,
        String countryName,
        boolean isActive
) {
}

File: frontend/models/address/AddressDetailVm.ts
export type AddressDetailVm = {
    id: number,
    contactName : string,
    phoneNumber: string,
    specificAddress: string,
    districtId: number,
    provinceId: number,
    countryId:number,
    // sau này có thể đưa về thêm tên
    districtName ?: string,
    provinceName?: string,
    countryName?: string,

    isActive?: boolean;

}